### PR TITLE
8.0 alpha branding

### PR DESCRIPTION
**Pull request by marcpopMSFT**

### CODE REVIEW

File: src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.SupportedTargetFrameworks.props [hunk 1]
```diff
@@ -25,6 +25,7 @@ Copyright (c) .NET Foundation. All rights reserved.
         <SupportedNETCoreAppTargetFramework Include=".NETCoreApp,Version=v5.0" DisplayName=".NET 5.0" Alias="net5.0" />
         <SupportedNETCoreAppTargetFramework Include=".NETCoreApp,Version=v6.0" DisplayName=".NET 6.0" Alias="net6.0" />
         <SupportedNETCoreAppTargetFramework Include=".NETCoreApp,Version=v7.0" DisplayName=".NET 7.0" Alias="net7.0" />
+        <SupportedNETCoreAppTargetFramework Include=".NETCoreApp,Version=v8.0" DisplayName=".NET 7.0" Alias="net8.0" />
```
Comment (danmoseley) [inline src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.SupportedTargetFrameworks.props:28]:
```suggestion
        <SupportedNETCoreAppTargetFramework Include=".NETCoreApp,Version=v8.0" DisplayName=".NET 8.0" Alias="net8.0" />
```
?

**Comment by kasperk81** [comment]
error is:
```
'System.InvalidOperationException: No instances of MSBuild could be detected.'
'Try calling RegisterInstance or RegisterMSBuildPath to manually register one.'
'   at Microsoft.Build.Locator.MSBuildLocator.RegisterDefaults()'
```

does bumping it to 1.5.3 fix the error: https://github.com/dotnet/sdk/blob/f19c3d78e26564e429c1236565a5e5fbdc154b14/eng/Versions.props#L170

**Review by RussKie — APPROVED** [review]

**Comment by marcpopMSFT** [comment]
I've rerun a bunch of times but there appear to be consistent failures in some of the watch tests. They only seem to fail on the release config of the full framework leg which is odd.

@tmat @arkalyanms, can you take a look?
```
System.NullReferenceException : Object reference not set to an instance of an object.


Stack trace
   at Microsoft.DotNet.Watcher.Tools.WatchableApp.GetProcessIdentifier() in /_/src/Tests/dotnet-watch.Tests/Utilities/WatchableApp.cs:line 75
   at Microsoft.DotNet.Watcher.Tools.NoDepsAppTests.RestartProcessThatTerminatesAfterFileChange() in /_/src/Tests/dotnet-watch.Tests/NoDepsAppTests.cs:line 59
--- End of stack trace from previous location ---
```

**Comment by tmat** [comment]
Addressed by https://github.com/dotnet/sdk/pull/27700

**Comment by tmat** [comment]
... which is failing with an unrelated intermittent issue https://github.com/dotnet/sdk/issues/27708

**Comment by marcpopMSFT** [comment]
@tmat I reran with the change from 27708 and had similar watch test failures. Can you take a look?  Are these at all possibly related to rebranding? I ask because we need this change in to unblock the installer repo but I don't want to merge and make main red.

**Comment by kasperk81** [comment]
underlying error is not nullref, it is same as https://github.com/dotnet/sdk/pull/27189#issuecomment-1233547068. scroll down on https://helix.dot.net/api/2019-06-17/jobs/16d3edd1-9224-4209-9b03-7e6e954ac695/workitems/dotnet-watch.Tests.dll.1/console and you will find `No instances of MSBuild could be detected`. @Forgind, @rainersigwald is it because of old azdo vm? this pr failed with both 1.4.1 and the latest 1.5.3

@marcpopMSFT, considering installer update is blocked for three weeks, how "important" is this dotnet-watch test blocking everyone? disabling this test and assigning the issue to area owner should be an option in such situations.

**Pull request merged**